packages/core/core-flows/src/order/workflows/create-return.ts

```diff
@@ -0,0 +1,305 @@
+import {
+  CreateOrderShippingMethodDTO,
+  FulfillmentWorkflow,
+  OrderDTO,
+  OrderWorkflow,
+  ShippingOptionDTO,
+} from "@medusajs/types"
+import {
+  createWorkflow,
+  transform,
+  WorkflowData,
+} from "@medusajs/workflows-sdk"
+import { createRemoteLinkStep, useRemoteQueryStep } from "../../common"
+import {
+  arrayDifference,
+  ContainerRegistrationKeys,
+  isDefined,
+  MathBN,
+  MedusaError,
+  Modules,
+} from "@medusajs/utils"
+import { updateOrderTaxLinesStep } from "../steps"
+import { createReturnStep } from "../steps/create-return"
+import { createFulfillmentWorkflow } from "../../fulfillment"
+
+function throwIfOrderIsCancelled({ order }: { order: OrderDTO }) {
+  // TODO: need work, check cancelled
+  if (false /*order.cancelled_at*/) {
+    throw new MedusaError(
+      MedusaError.Types.INVALID_DATA,
+      `Order with id ${order.id} has been cancelled.`
+    )
+  }
+}
+
+function throwIfItemsDoesNotExistsInOrder({
```

Comment:
thought: This could probably be a more generic helper – I imagine we will need this elsewhere

Reply:
yes definately, for now they are local, but as soon as we have other needs then we can move them to utils wdyt?